extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -411,6 +414,10 @@ public static function closeAccount( $user = '', $changeReason = '', &$mStatusMs
 		// if they are connected from facebook, disconnect them
 		self::disconnectFromFacebook( $user );
 
+		/** @var HeliosClient $heliosClient */
+		$heliosClient = Injector::getInjector()->get(HeliosClient::class);
+		$heliosClient->forceLogout($user->getId());
```

Comment:
If it fails, it will be notice in kibana.
I asked in ticket for expecting behavior, but at this moment, everything will works as was before this commit with additional call to helios.